modules/RouterUtils.js

```diff
@@ -1,26 +1,7 @@
-import deprecateObjectProperties from './deprecateObjectProperties'
-
 export function createRouterObject(history, transitionManager) {
   return {
     ...history,
     setRouteLeaveHook: transitionManager.listenBeforeLeavingRoute,
```

Comment:
It's always been the transition manager's `listen`. A while ago I'd wanted to rename it to `listenRoutes`, but it's not so important now that it's called `router.listen` instead of `history.listen`.